master/pkg/schemas/expconf/hyperparameters_config.go

```diff
@@ -22,6 +47,7 @@ type HyperparameterV0 struct {
 	RawDoubleHyperparameter      *DoubleHyperparameterV0      `union:"type,double" json:"-"`
 	RawLogHyperparameter         *LogHyperparameterV0         `union:"type,log" json:"-"`
 	RawCategoricalHyperparameter *CategoricalHyperparameterV0 `union:"type,categorical" json:"-"`
+	RawNestedHyperparameter      *map[string]HyperparameterV0 `union:"type,object" json:"-"`
 }
 
 // Merge prevents recursive merging of hyperparameters.
```

Comment:
I actually think it might be reasonable to allow the NestedHyperparameter should be merged recursively.  What do you think?